src/mono/browser/test-main.js

```diff
@@ -24,6 +24,7 @@ export const ENVIRONMENT_IS_SIDECAR = ENVIRONMENT_IS_WEB_WORKER && typeof dotnet
 export const ENVIRONMENT_IS_WORKER = ENVIRONMENT_IS_WEB_WORKER && !ENVIRONMENT_IS_SIDECAR; // we redefine what ENVIRONMENT_IS_WORKER, we replace it in emscripten internals, so that sidecar works
 export const ENVIRONMENT_IS_WEB = typeof window == "object" || (ENVIRONMENT_IS_WEB_WORKER && !ENVIRONMENT_IS_NODE);
 export const ENVIRONMENT_IS_SHELL = !ENVIRONMENT_IS_WEB && !ENVIRONMENT_IS_NODE;
+export const isFirefox = ENVIRONMENT_IS_WEB && navigator.userAgent.includes("Firefox");
```

Comment:
`// keep in sync with src\mono\browser\runtime\loader\globals.ts and src\mono\browser\runtime\globals.ts`
do we treat it as binding for variables not beginning with `ENVIRONMENT`?

Reply:
those `ENVIRONMENT`* things are kin to emscripten for the same thing.